noetic/distribution.yaml

```diff
@@ -1091,14 +1091,30 @@ repositories:
     doc:
       type: git
       url: https://github.com/4am-robotics/cob_common.git
-      version: kinetic_release_candidate
+      version: noetic-release-candidate
     release:
       packages:
       - cob_actions
       - cob_common
-      - cob_description
       - cob_msgs
       - cob_srvs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/4am-robotics/cob_common-release.git
```

Comment:
Also, https://github.com/4am-robotics/cob_common-release/blob/14d26789362ce411824cdcfc0099328475d50e41/tracks.yaml#L139 uses `noetic-release-candidate` branch to release from.  Most likely, all of `doc`, `source`, and the `devel_branch` in tracks.yaml should be the same.  At least, that will make things the most understandable.

Reply:
I actually updated the doc (noetic-release-candidate) and source (noetic-devel) branch to match the new devel (noetic-devel) branch for the packages released with version 0.7.11 under the cob_common entry

Or do you mean why a *_release_candidate branch is used for doc and a *-devel branch for source?
The latter, I actually dont know myself, I just kept to the scheme used before, i.e. kinetic_release_candidate for doc and kinetic_dev for source

If you ask me to change this, I'd rather do this in a follow-up PR that changes it consistebtly for all cob_* repos - as it might affect more repos beyond the 4 repos that I updated in the current cycle

What do you think?

Reply:
@clalancette friendly ping

Reply:
anything I can/should do here for now?

Reply:
> Or do you mean why a *_release_candidate branch is used for doc and a *-devel branch for source? The latter, I actually dont know myself, I just kept to the scheme used before, i.e. kinetic_release_candidate for doc and kinetic_dev for source It has probably been like this doc vs. source for years now...
> 
> If you ask me to change this, I'd rather do this in a follow-up PR that changes it consistebtly for all cob_* repos - as it might affect more repos beyond the 4 repos that I updated in the current cycle

All right, that is fine.  But I do encourage you to update it; like I said, it is somewhat unusual for the doc and source branches to be different.